Project.toml

```diff
@@ -16,7 +16,7 @@ SolverTools = "b5612192-2639-5dc1-abfe-fbedd65fab29"
 
 [compat]
 Krylov = "0.9.6"
-LinearOperators = "2.0"
+LinearOperators = "2.9"
```

Comment:
Can you tell us why we need this?

Reply:
I am using the solve_shifted_system! method which is added to 2.9